runners/spark/job-server/container/build.gradle

```diff
@@ -54,9 +54,10 @@ task copyDockerfileDependencies(type: Copy) {
 }
 
 docker {
-  name containerImageName(name: 'spark-job-server',
+  name containerImageName(name: ${project.docker_image_default_repo_prefix} + 'job-server',
```

Comment:
Looks like we lost `spark` here.

Reply:
Fix it.
Is there any reason we used `-` instead of `_` for `spark-job-server`?

Reply:
I suppose it should probably use underscores like everywhere else.